features/features.go

```diff
@@ -70,6 +70,8 @@ const (
 	QuotaGroups
 	// RefreshAppAwarenessUX enables experimental UX improvements for refresh-app-awareness.
 	RefreshAppAwarenessUX
+	// AspectsConfiguration enables experimental aspect-based configuration.
+	AspectsConfiguration
```

Comment:
do we need tests in features_test.go for this? does that need to add count checks to force that?